deploy/dev/config/k8s/init-solution.sh.yml

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
thoughts on the file name. i understand that you chose it to say that this is the secrets-provider init container solution but it is not clear imo. it can be understood as "init the solution". if i understand correctly, you plan to add in the future files like `job-solution`, `rotation-solution`, etc. 

i think that we should choose another pattern that will be more self explanatory, such as `secrets-provider-init-container`, `secrets-provider-job`, etc.